x12-edi/2.6/modules/ROOT/pages/x12-edi-connector-batching.adoc

```diff
@@ -0,0 +1,60 @@
+= X12 EDI Connector 2.3 - Batch Outbound X12 Messages - Mule 4
+
+For EDI messages that are not time-sensitive, such as invoice and payment remittances, you can use the Batch Write operation to merge multiple individual messages into a single interchange to send to your trading partners.
```

Comment:
Please duplicate the edits made to the same text in edifact-edi/2.3/modules/ROOT/pages/edifact-edi-connector-batching.adoc (applying the proper connector name, of course).
I will scan to look for anything that is different....